mex/common/primary_source/helpers.py

```diff
@@ -0,0 +1,46 @@
+from functools import cache
+
+from mex.common.models import (
+    ExtractedPrimarySource,
+)
+from mex.common.primary_source.extract import extract_seed_primary_sources
+from mex.common.primary_source.transform import (
+    transform_seed_primary_sources_to_extracted_primary_sources,
+)
+
+### this code is nearly the same as in the pytest.fixture.extracted_primary_sources.
+### Can we reuse this?
```

Comment:
yep, let's do that. the other way round though, using this function within the testing.plugin (or maybe that's also what you meant?)